### PR TITLE
Added InventoryErrors, SCP_Interfaces and updated all the other files for FI

### PR DESCRIPTION
**Pull request by CallMeBlad3z**
# SCP:SL Translation PR Template

## Language
fi

## Type of change
- [ ] Added new language translation
- [x] Changed a currently existing translation

## Have you tested the changes you've made?
- [x] Yes
- [ ] No

# Checklist:
- [x] I have made sure to not commit changes to other stuff than my own or the default english one
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x] I have made sure that I'm not submitting a duplicate translation 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of new translations

### CODE REVIEW

**Comment by MichalPetryka** [comment]
Please run TranslationChecker and see the results

**Comment by CallMeBlad3z** [comment]
Done. Added 3 files and updated the remaining ones.

File: Translations/fi/MainMenu.txt [hunk 1]
```diff
@@ -59,4 +59,29 @@ ENIMMÄIS VEREN MÄÄRÄ
...
+Me käytämme Steamin foorumia keskustelualueenamme. (Paina logoa avataksesi foorumin)
+ÄÄNI ASETUKSET
+MASTER VOLUME
+PÄÄVALIKKO MUSIIKKI
+ÄÄNIKESKUSTELU
+ÄÄNITEHOSTEET
+PELIN VAIHTOEHDOT
+NOPEA ESITTELYN HAIHTUMINEN
+Luokan esittely teksti katoaa paljon nopeammin.
+SCP-079 - VAIHDA NÄHTÄVYYS TILAA
```
Comment (Dogera) [inline Translations/fi/MainMenu.txt]: "Nähtävyys" -> "Näkyvyys"?

File: Translations/fi/MainMenu.txt [hunk 2]
```diff
@@ -59,4 +59,29 @@ ENIMMÄIS VEREN MÄÄRÄ
...
+Paina kerran vaihtaaksesi kamera tilaa.
```
Comment (Dogera) [inline Translations/fi/MainMenu.txt]: ...kamera**n** tilaa?

File: Translations/fi/MainMenu.txt [hunk 3]
```diff
@@ -59,4 +59,29 @@ ENIMMÄIS VEREN MÄÄRÄ
...
+SCP-079 - JÄLKIKÄSITTELY
+Jos poissa käytöstä, kameran tehosteet vähenevät huomattavasti.
```
Comment (Dogera) [inline Translations/fi/MainMenu.txt]: "kameran" -> "näytön"?

File: Translations/fi/MainMenu.txt [hunk 4]
```diff
@@ -59,4 +59,29 @@ ENIMMÄIS VEREN MÄÄRÄ
...
+PÄÄN KEIKKUMINEN
+Aktivoi ensimmäisen persoonan kameran keikkumisen liikkeessä.
+AKTIVOI JUOKSU
+Sprinting toggles on and off, instead of being held down.
```
Comment (Dogera) [inline Translations/fi/MainMenu.txt]: "Juokseminen vaihtelee päälle ja pois, napin pohjassapitämisen sijaan." Unsure why not translated

File: Translations/fi/InventoryErrors.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<color=yellow>{0} ammusten</color> raja saavutettu (<color=yellow>{1} ammusta</color>)
+Voit kantaa <color=yellow>{0}</color> tämän tyyppistä tavaraa (<color=yellow>{1}</color>)
```
Comment (Dogera) [inline Translations/fi/InventoryErrors.txt]: "Voit kantaa **vain** tämän..."?

File: Translations/fi/SCP_Interfaces.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+Tämä ihmisparka on rappeutunut parannuksen jälkeen.
```
[the remark this answers is not in the document]
Comment (CallMeBlad3z) [reply Translations/fi/SCP_Interfaces.txt]: The thing is i don't know where it shows this in the game.

**Comment by CallMeBlad3z** [comment]
Everything should be fine now.

**Pull request merged**